src/Uplink/Config.php

```diff
@@ -31,6 +31,27 @@ class Config {
 	 */
 	protected static $hook_prefix = '';
 
+	/**
+	 * Whether your plugin allows multisite network subfolder licenses.
+	 *
+	 * @var bool
+	 */
+	protected static $network_subfolder_license = false;
```

Comment:
The `::reset()` method needs to reset all of the static var values back to defaults. (just commenting here, though the comment applies to all of the static vars.)